framework/mesh/mesh.cc

```diff
@@ -1,40 +1,22 @@
 #include "framework/mesh/mesh.h"
-#include "framework/mesh/mesh_handler/mesh_handler.h"
 #include "framework/runtime.h"
 #include "framework/mesh/unpartitioned_mesh/unpartitioned_mesh.h"
 #include "framework/logging/log.h"
 
 namespace opensn
 {
 
-MeshHandler&
-GetCurrentHandler()
+std::shared_ptr<MeshContinuum>
+GetCurrentMesh()
 {
-  if (meshhandler_stack.empty())
-    throw std::logic_error("chi_mesh::GetCurrentHandler: No handlers on stack");
-
-  return GetStackItem<MeshHandler>(meshhandler_stack, current_mesh_handler);
-}
-
-size_t
-PushNewHandlerAndGetIndex()
-{
-  meshhandler_stack.push_back(std::make_shared<MeshHandler>());
-
-  int index = (int)meshhandler_stack.size() - 1;
-  current_mesh_handler = index;
-
-  return index;
```

Comment:
what are the implications of this? only one mesh can be "live" at a time during execution? @zhardy-lanl for opinion. This looks simpler but I just want to be sure.

Reply:
There is this now: `std::vector<std::shared_ptr<MeshContinuum>> mesh_stack`. This serves as a stack, so the lua layer should be able to grab any mesh from the stack and use it. But, right now, as @zhardy-lanl said, there is only one element in the vector.